Pages/wishlist.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>E-Yard | My Wishlist</title>
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.12.1/css/all.css"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="../CSS/CommonStyles/style.css" />
+    <link rel="stylesheet" href="../CSS/wishlist.css" />
```

Comment:
Is it possible to import wishlist.css to style.css. It looks like your style.css is your main css file.